labs/lab4.md

```diff
@@ -3,9 +3,22 @@
 Useful links:
 
 * <https://www.elastic.co/guide/en/elasticsearch/reference/current/inference-processor.html>
+* <https://www.elastic.co/guide/en/machine-learning/master/ml-nlp.html>
+* <https://www.elastic.co/guide/en/machine-learning/master/ml-nlp-ner-example.html>
+* <https://www.elastic.co/guide/en/machine-learning/master/ml-nlp-text-emb-vector-search-example.html>
 
-## Steps
+## Use case 1: Named Entity Recognition (NER) Model
+### Steps
+- Deploy a named entity recognition (NER) model in Elasticsearch
```

Comment:
I'd add some further information. At least "using eland" or even better, we can give them the command line to run as it's a bit complex. WDYT? That'd be aligned with https://github.com/dadoonet/DevoxxFR-2023/blob/main/labs/lab5.md

Reply:
I agree. I will add the part about cloning Eland and creating the Docker image. also the deployment to the Elastic cloud.